src/deadline/client/ui/dialogs/submit_job_progress_dialog.py

```diff
@@ -361,3 +349,113 @@ def _build_ui(self):
 
         self.layout.addWidget(self.progress_bar)
         self.layout.addWidget(self.progress_message)
+
+
+class ScrollableWarningDialog(QDialog):
+    """
+    Simple Dialog which functions similar to a QMessageBox, but with a scrollable text area.
+    """
+
+    def __init__(self, message: str, default_response: bool = False):
```

Comment:
nit: minor usability improvement to the caller IMO

```suggestion
    def __init__(self, message: str, default_button: QDialogButtonBox = QDialogButtonBox.Cancel):
```

Reply:
Got it. It is less of a concern given it has been changed to a private API. Thanks